shub/login.py

```diff
@@ -1,21 +1,56 @@
-import os, re, click
-from shub.utils import get_key_netrc, NETRC_FILE
+import re
+import click
+import os
+from shub import auth
+from scrapy.utils.conf import get_config
```

Comment:
Good catch @eliasdorneles!

Looking for `scrapy.cfg` file should only happen if scrapy is available, but otherwise it should still work (not support reading API keys from scrapy.cfg).

The tests should run without `scrapy` installed, otherwise it will be hard to keep this somewhat intricate logic working.